smuggler/utils.py

```diff
@@ -8,10 +8,13 @@
 from django.core.management.color import no_style
 from django.core.management.commands.dumpdata import Command as DumpData
 from django.core.management.commands.loaddata import Command as LoadData
+from django.core.management import call_command
+
 from django.db.utils import DEFAULT_DB_ALIAS
 from django.http import HttpResponse
 from django.utils.six import StringIO
 from smuggler import settings
+import django
```

Comment:
Put the import statement on top. I would also suggest a line space between `from smuggler...` and `from django... *` stuff.

That is a recommendation from PEP 8:

> Imports should be grouped in the following order:
> 
> 1. standard library imports
> 2. related third party imports
> 3. local application/library specific imports
> 4. You should put a blank line between each group of imports.

The same applies to the changes at file _tests/test_app/tests/test_dump.py_.